pybotters/models/gmocoin.py

```diff
@@ -0,0 +1,616 @@
+            else None,
+            order_timestamp=parse_datetime(data["orderTimestamp"])
+            if "orderTimestamp" in data
+            else None,
+            time_in_force=data.get("timeInForce", None),
+        )
+
+    @staticmethod
+    def to_orders(data: List[Item]) -> List["Order"]:
+        return [MessageHelper.to_order(x) for x in data]
+
+    @staticmethod
+    def to_order(data: Item) -> "Order":
+        status = OrderStatus[data.get("status", data.get("orderStatus"))]
+        timestamp = parse_datetime(data.get("orderTimestamp", data.get("timestamp")))
+        return Order(
+            order_id=data["orderId"],
+            symbol=Symbol[data["symbol"]],
+            settle_type=SettleType[data["settleType"]],
+            execution_type=ExecutionType[data["executionType"]],
+            side=OrderSide[data["side"]],
+            order_status=status,
+            cancel_type=CancelType[data.get("cancelType", CancelType.NONE.name)],
+            order_timestamp=timestamp,
+            price=int(data.get("price", data.get("orderPrice"))),
+            size=Decimal(data.get("size", data.get("orderSize"))),
+            executed_size=Decimal(
+                data.get("executedSize", data.get("orderExecutedSize"))
+            ),
+            losscut_price=int(data["losscutPrice"]),
+            time_in_force=data["timeInForce"],
+        )
+
+    @staticmethod
+    def to_positions(data: List[Item]) -> List["Position"]:
+        return [MessageHelper.to_position(x) for x in data]
+
+    @staticmethod
+    def to_position(data: Item) -> "Position":
+        return Position(
+            position_id=data["positionId"],
+            symbol=Symbol[data["symbol"]],
+            side=OrderSide[data["side"]],
+            size=Decimal(data["size"]),
+            orderd_size=Decimal(data["orderdSize"]),
+            price=int(data["price"]),
+            loss_gain=int(data["lossGain"]),
+            leverage=Decimal(data["leverage"]),
+            losscut_price=int(data["losscutPrice"]),
+            timestamp=parse_datetime(data.get("timestamp")),
+        )
+
+    @staticmethod
+    def to_position_summaries(data: List[Item]) -> List["PositionSummary"]:
+        return [MessageHelper.to_position_summary(x) for x in data]
+
+    @staticmethod
+    def to_position_summary(data: Item) -> "PositionSummary":
+        return PositionSummary(
+            symbol=Symbol[data["symbol"]],
+            side=OrderSide[data["side"]],
+            average_position_rate=int(data["averagePositionRate"]),
+            position_loss_gain=int(data["positionLossGain"]),
+            sum_order_quantity=Decimal(data["sumOrderQuantity"]),
+            sum_position_quantity=Decimal(data["sumPositionQuantity"]),
+            timestamp=parse_datetime(data.get("timestamp"))
+            if data.get("timestamp")
+            else datetime.now(timezone.utc),
+        )
+
+
+class GmoCoinDataStoreInterface(DataStoreInterface):
```

Comment:
```suggestion
class GMOCoinDataStore(DataStoreInterface):
```
`Interface` を外して頂きたいのと、
#69 よりPythonの正しい命名規則ではないですが`Gmo` -> `GMO`でお願いできますか？